views/index.py

```diff
@@ -84,14 +86,23 @@ def registration():
         form = RegistrationForm()
         if form.validate_on_submit():
             try:
-                new_user = User(
-                    email=form.email.data,
-                    first_name=form.first_name.data,
-                    surname=form.surname.data,
-                    password_hash=generate_password_hash(form.password.data))
-                db.session.add(new_user)
-                db.session.commit()
-                send_confirmation_email(new_user.email)
+                data = User.query.filter_by(email=form.email.data).first()
+                if data is None:
+                    new_user = User(
+                        email=form.email.data,
+                        first_name=form.first_name.data,
+                        surname=form.surname.data,
+                        password_hash=generate_password_hash(
+                            form.password.data))
+                    db.session.add(new_user)
+                    db.session.commit()
+                    send_confirmation_email(new_user.email)
+                else:
+                    message_body = 'User already exist'
+                    message_title = 'Opss!'
+                    return render_template('message.html',
```

Comment:
I think we can move message_body and title to the static property in RegisterForm. Later we can use:
message_title=RegistrationForm.error_title etc.